stable/datadog/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: datadog
-version: 0.11.2
+version: 0.11.3
```

Comment:
As you're adding new features, please bump the minor version of the Chart.

Reply:
Yes, minor bump, please